eval.sh

```diff
@@ -2,7 +2,6 @@
 mkdir -p ./eval/
 python benchmark.py --gold=./oie_corpus/all.oie --out=eval/Stanford.dat --stanford=./systems_output/stanford_output.txt
 python benchmark.py --gold=./oie_corpus/all.oie --out=eval/OLLIE.dat --ollie=./systems_output/ollie_output.txt
-python benchmark.py --gold=./oie_corpus/all.oie --out=eval/ReVerb.dat --reverb=./systems_output/reverb_output.txt
```

Comment:
ReVerb breaks when this bug is fixed. It seems like the `reverb_output.txt` file is actually broken. The sentence IDs don't correspond right to the sentence IDs in the ReVerb corpus. Temporarily removing this from `eval.sh` until it can be fixed. This shouldn't be a problem since ReVerb is predecessor to some of the later work like OpenIE-4?

Reply:
Oh, let me take a look at this. 
Sounds good to defer this for now, I can open an issue if something's broken.